src/wifi.rs

```diff
@@ -1615,7 +1615,8 @@ pub struct ApProbeRequestReceivedData {
 pub enum WifiEvent {
     Ready,
 
-    ScanDone,
+    ScanStarted,
```

Comment:
ScanStarted seems to have been removed upstream and erroneously reintroduced here https://github.com/Stabl-Energy/esp-idf-svc/compare/423419acf77b7990a99311b9efeda5e404ac0f35...Stabl-Energy:esp-idf-svc:feature/extended-wifi-event#diff-163838ea4fbc63179426321ccf5a036e1c4cbdd36adcdde9a9e30dbf5ae08c4fL1077-L1079

Reply:
I have removed the variant.